.claude/commands/digissue.md

```
@@ -0,0 +1,174 @@
+コマンド引数には `{Issue番号}` が渡される。
+- 例: `/digissue 123`
+- Issue番号がない場合は、オープン中のIssue一覧を表示して選択を促す。
+
+## 処理フロー
+
+### 1. Issue情報の取得
+```bash
+gh issue view {Issue番号} --json title,body,labels,assignees,milestone,comments
+```
+- Issueのタイトル、本文、ラベル、担当者、マイルストーン、**コメント**を全て取得する
+- **重要: Issueのコメントは全て取得して読むこと**（追加のログや情報が含まれている可能性がある）
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)